### ISSUE TITLE
bcbio installation on centos 7.6

### ISSUE THREAD

**Issue by dauss75** [bcbio/bcbio-nextgen#2589]
Hello,

On CentOS 7.6, I am trying to install bcbio with following command: 
$python bcbio_nextgen_install.py /mnt/bcbio-data --tooldir=/rabbit/bcbio -genomes GRCh37 --aligners bwa --aligners bowtie2

Then, it looked all good installing until I get to this point:

2018-12-07 14:56:40 (402 KB/s) - ‘coverage/problem_regions/repeats/polyx.bed.gz.tbi’ saved [355665/355665]

Connecting to nc.hartwigmedicalfoundation.nl.x.86bbc427022ad04c700a0c400943b6eb6d9d.ccc2eda1.id.opendns.com|204.194.237.161|:443... connected.
WARNING: cannot verify nc.hartwigmedicalfoundation.nl.x.86bbc427022ad04c700a0c400943b6eb6d9d.ccc2eda1.id.opendns.com's certificate, issued by ‘O=Cisco,CN=Cisco Umbrella Secondary SubCA lax-SG’:
  Unable to locally verify the issuer's authority.
HTTP request sent, awaiting response... 302 Moved Temporarily
Location: https://bpb.opendns.com/b/https/nc.hartwigmedicalfoundation.nl/index.php/s/a8lgLsUrZI5gndd/download?X-OpenDNS-Session=aa51dc5c0ccc5aad4_86bbc427022ad04c700a0c400943b6eb6d9d_bdb6e5698e8bddf337724690f430e7d26ceaa1e2_path=%2FHMF-Pipeline-Resources&files=GC_profile.hg19.1000bp.cnp.gz [following]
--2018-12-07 14:56:40--  https://bpb.opendns.com/b/https/nc.hartwigmedicalfoundation.nl/index.php/s/a8lgLsUrZI5gndd/download?X-OpenDNS-Session=aa51dc5c0ccc5aad4_86bbc427022ad04c700a0c400943b6eb6d9d_bdb6e5698e8bddf337724690f430e7d26ceaa1e2_path=%2FHMF-Pipeline-Resources&files=GC_profile.hg19.1000bp.cnp.gz
Connecting to bpb.opendns.com|204.194.237.163|:443... connected.
HTTP request sent, awaiting response... 302 Moved Temporarily
Location: /b/https/nc.hartwigmedicalfoundation.nl/index.php/s/a8lgLsUrZI5gndd/download?path=%2FHMF-Pipeline-Resources&files=GC_profile.hg19.1000bp.cnp.gz [following]
--2018-12-07 14:56:40--  https://bpb.opendns.com/b/https/nc.hartwigmedicalfoundation.nl/index.php/s/a8lgLsUrZI5gndd/download?path=%2FHMF-Pipeline-Resources&files=GC_profile.hg19.1000bp.cnp.gz
Reusing existing connection to bpb.opendns.com:443.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [text/html]
Saving to: ‘STDOUT’
 
-                         [ <=>                     ]   6.35K  --.-KB/s    in 0s
 
2018-12-07 14:56:40 (57.0 MB/s) - written to stdout [6507]
 
 
gzip: stdin: not in gzip format
Traceback (most recent call last):
  File "/mnt/bcbio-data/anaconda/bin/bcbio_nextgen.py", line 221, in <module>
    install.upgrade_bcbio(kwargs["args"])
  File "/mnt/bcbio-data/anaconda/lib/python2.7/site-packages/bcbio/install.py", line 106, in upgrade_bcbio
    upgrade_bcbio_data(args, REMOTES)
  File "/mnt/bcbio-data/anaconda/lib/python2.7/site-packages/bcbio/install.py", line 344, in upgrade_bcbio_data
    cbl_deploy.deploy(s)
  File "/home/sequencer/tmpbcbio-install/cloudbiolinux/cloudbio/deploy/__init__.py", line 63, in deploy
    _setup_vm(options, vm_launcher, actions)
  File "/home/sequencer/tmpbcbio-install/cloudbiolinux/cloudbio/deploy/__init__.py", line 109, in _setup_vm
    configure_instance(options, actions)
  File "/home/sequencer/tmpbcbio-install/cloudbiolinux/cloudbio/deploy/__init__.py", line 269, in configure_instance
    setup_biodata(options)
  File "/home/sequencer/tmpbcbio-install/cloudbiolinux/cloudbio/deploy/__init__.py", line 251, in setup_biodata
    install_proc(options["genomes"], ["ggd", "s3", "raw"])
  File "/home/sequencer/tmpbcbio-install/cloudbiolinux/cloudbio/biodata/genomes.py", line 340, in install_data
    _prep_genomes(env, genomes, genome_indexes, ready_approaches)
  File "/home/sequencer/tmpbcbio-install/cloudbiolinux/cloudbio/biodata/genomes.py", line 469, in _prep_genomes
    retrieve_fn(env, manager, gid, idx)
  File "/home/sequencer/tmpbcbio-install/cloudbiolinux/cloudbio/biodata/genomes.py", line 836, in _install_with_ggd
    ggd.install_recipe(env.cwd, recipe_file, gid)
  File "/home/sequencer/tmpbcbio-install/cloudbiolinux/cloudbio/biodata/ggd.py", line 31, in install_recipe
    recipe["recipe"]["full"]["recipe_type"])
  File "/home/sequencer/tmpbcbio-install/cloudbiolinux/cloudbio/biodata/ggd.py", line 63, in _run_recipe
    subprocess.check_output(["bash", run_file])
  File "/mnt/bcbio-data/anaconda/lib/python2.7/subprocess.py", line 223, in check_output
    raise CalledProcessError(retcode, cmd, output=output)
subprocess.CalledProcessError: Command '['bash', '/mnt/bcbio-data/genomes/Hsapiens/GRCh37/txtmp/ggd-run.sh']' returned non-zero exit status 1
Traceback (most recent call last):
  File "bcbio_nextgen_install.py", line 265, in <module>
    main(parser.parse_args(), sys.argv[1:])
  File "bcbio_nextgen_install.py", line 44, in main
    subprocess.check_call([bcbio, "upgrade"] + _clean_args(sys_argv, args))
  File "/usr/lib64/python2.7/subprocess.py", line 542, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/mnt/bcbio-data/anaconda/bin/bcbio_nextgen.py', 'upgrade', '--tooldir=/rabbit/bcbio', '--genomes', 'GRCh37', '--aligners', 'bwa', '--aligners', 'bowtie2', '--data']' returned non-zero exit status 1

-----
Do you have any clue on what would possibly be causing this issue?
I find some posts relating to Cisco root CA, but not sure what to do yet.

Many thanks!
Chris

**Comment by dauss75** [comment]
OK.  I think the website is blocked by our IT and they are investigating.  Will update if that's not the case.

**Comment by dauss75** [comment]
Thanks for the reply.  Our IT got it right.  
Now, I have a different issue that one of the servers is not responding as shown below. 

Running GGD recipe: GRCh37 hapmap 3.3
--2018-12-10 12:54:13--  ftp://gsapubftp-anonymous:*password*@ftp.broadinstitute.org/bundle/b37/hapmap_3.3.b37.vcf.gz
           => ‘-’
Resolving ftp.broadinstitute.org... 69.173.80.251
Connecting to ftp.broadinstitute.org|69.173.80.251|:21... connected.
Logging in as gsapubftp-anonymous ...
Error in server response, closing control connection.
Retrying.

**Comment by chapmanb** [comment]
Glad the previous problem is solved and thanks for following up. Unfortunately, the Broad FTP server is commonly oversubscribed and reports these type of login issues when that happens. If you re-try when this happens it should hopefully be less busy and work correctly. Hope this works for you.

**Comment by dauss75** [comment]
Trying again this morning and it's same.  Any chance that the server is down?  I wonder if there is work-around to download the files.

**Comment by dauss75** [comment]
That was another issue with our IT department so I think we can close this.

**Comment by chapmanb** [comment]
Thanks much for following up, and glad you're able to work through the issues. Hopefully you get the install finished cleanly and can work on running actual analyses.
